recipes/signac/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "signac" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev: v{{ version }}
+  git_url: https://bitbucket.org/glotzer/signac
```

Comment:
I'm not sure if there was some failure in the feedstock creation or if it is stuck in a CI queue. @jakirkham -- is there a simple way to figure this out?